authors/apps/authentication/views.py

```diff
@@ -10,7 +10,7 @@
 )
 
 
-class RegistrationAPIView(APIView):
+class RegistrationAPIView(GenericAPIView):
```

Comment:
@calvinpete Why the change to using `GenericAPIView`.

Reply:
The Input parameters do not appear in the swagger documentation when we use APIView but the GenericAPIView uses the get_serializer class to implement the views